### DISCUSSION TITLE
Notification

### DISCUSSION THREAD

**Discussion by Amir999** [directus/directus#15240] [Archive]
Hi everyone;
I would like to know how to enable notifications on the directus

Cordially

**Comment by rijkvanzanten** [comment]
Hi @Amir999! We only operate the repository and its discussions/issues in English. Apologies for the inconvenience.

**Reply by Amir999** [reply]
Sorry for this mistake, i change it now

**Comment by rijkvanzanten** [comment]
What do you mean by _enable notifications_? Are you looking to send yourself an email on an item change for example?

**Reply by Amir999** [reply]
yes, but send an email automatically

**Reply by rijkvanzanten** [reply]
Gotcha! You can use Flows to setup a workflow that will automatically send an email whenever an item changes based on conditions. @erondpowell is this one of the examples we've got cooking up?

**Reply by Amir999** [reply]
Thank you, if you have an example i'll be happy @rijkvanzantenand and @erondpowell  :-)

**Reply by erondpowell** [reply]
> Gotcha! You can use Flows to setup a workflow that will automatically send an email whenever an item changes based on conditions. @erondpowell is this one of the examples we've got cooking up?

Yes, we'll have a solid example of every single flow trigger and operation.

@Amir999 We don't have this written up yet but it will be here within a few weeks... Have you tried to make an email operation in a flow yet?

**Reply by Amir999** [reply]
@erondpowell Yes i have : 
![Capture](https://user-images.githubusercontent.com/6806283/189840479-24bbb65c-8aad-4f6c-bdc6-8a33bb896c88.PNG)
![Capture2](https://user-images.githubusercontent.com/6806283/189840484-2ba7fe20-002a-48bc-9ee6-7b696771875c.PNG)

**Reply by erondpowell** [reply]
@Amir999 Nice flow! Three quick things:

**Can you show the log produced when you trigger the flow?**
There are a lot of configuration options in the triggers and flows.
One thing I notice is that you don't have a log function, so we can't be sure if the point of failure is at the `trigger` or `operation`.
Using the `log` operation and the Activity Log will be very helpful in pinpointing where the problem begins.

**Can you show me how it was configured?**
This flow is pretty simple. If the problem is in the Flows, there are only two points of failure: the trigger or email.
Can you click to open the trigger and operation panels and let me see how each was configured?

**Are you running this project locally?**
Check the spam folder of your email account. If the flow email came from  `localhost` your email provider may flag it as suspicious.
